html-block-and-inline/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
+    <p>Part of our new Americana collection, this <i>beautiful</i> blue is already a customer favorite!</p>
```

Comment:
Although `<i>` works here, it is best to use `<em>` for italics. The `<em>` tag is recognized by screen reader whereas `<i>` tags are not.